modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc

```diff
@@ -0,0 +1,59 @@
+= Accessing the console
+
+The {app-security-service} Console is viewed by opening the console URL in a browser. This URL is seen in the list of available services in the Mobile Developer Console (MDC) after the {app-security-service} Service has been provisioned.
+
+= Console overview
+
+The {app-security-service} Console allows you to monitor applications, their respective versions, and disable versions of those applications.
+
+Opening the console presents the applications view. Clicking on an application opens a details view for that application.
```

Comment:
The console lists all the applications <condition?.
Click on the name of an application to view the details for that application.